projects/storefrontlib/src/cms-components/configurator/commons/to-configuration/to-configuration.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from '@angular/core';
+import { RoutingService } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-to-configuration',
+  templateUrl: './to-configuration.component.html',
+})
+export class ToConfigurationComponent {
```

Comment:
`cx-to-configuration` looks a bit ambigious IMHO. What do you think about `cx-configure` component?